### PR TITLE
[IMP] version: semver up to 2.13.0

### PR DESCRIPTION
**Pull request by SButko**
Update semver package to latest stable version.

version.py: fix depreciation warnings for methods which are deprecated
since version 2.10.0.
Docs: https://python-semver.readthedocs.io/_/downloads/en/2.10.2/pdf/

[BRANCH] feature/semver-update-sbu

### CODE REVIEW

**Comment by oerp-odoo** [comment]
I thought 3.0.0 was released, but it is still in development. OK, so lets use this now.

File: tox.ini [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+[testenv]
+extras = test
+commands = py.test {posargs}
+
+[testenv:lint]
+basepython = python3.9
+skip_install=true
+deps =
+    flake8
```
Comment (AurelijaNorvaisaite) [inline tox.ini]: Is it necessary to write in newline?

Comment (SButko) [reply tox.ini]: Not sure it's necessary, though it's (almost) copy-paste from `maraplus` repository and it was introduced by Andrius. I guess, he used such format on purpose.

Comment (oerp-odoo) [reply tox.ini]: It should not be necesary if there is one line. I think there were multiple lines and then I removed one.

**Review by oerp-odoo — APPROVED** [review]

**Pull request merged**